src/scythe-bidder/create-room.tsx

```diff
@@ -2,31 +2,83 @@
 
 import React from "react";
 import { jsx } from "@emotion/core";
-import { Button, Card, Form, notification, Select, Switch } from "antd";
+import { Button, Card, Form, notification, Select, Popover } from "antd";
 import client from "./client";
-import { FACTIONS_BASE, FACTIONS_IFA, MATS_BASE, MATS_IFA } from "./constants";
+import {
+  FACTIONS_BASE,
+  FACTIONS_IFA,
+  FACTIONS_HI,
+  FACTIONS_LO,
+  MATS_BASE,
+  MATS_IFA,
+  MATS_HI,
+  MATS_LO,
+} from "./constants";
 import { MAX_PLAYERS_BASE, MAX_PLAYERS_IFA, MIN_PLAYERS } from "./constants";
 import { SCYTHE_BIDDER } from "./constants";
+import { QuestionCircleFilled } from "@ant-design/icons";
 
 export default function CreateRoom({ onCreate }: { onCreate: () => void }) {
   const [numPlayers, setNumPlayers] = React.useState(2);
   const [isIfaActive, setIsIfaActive] = React.useState<boolean>(true);
+  const [activeCombinations, setActiveCombinations] = React.useState<string>(
+    "IFA"
+  );
   const maxPlayers = isIfaActive ? MAX_PLAYERS_IFA : MAX_PLAYERS_BASE;
 
+  const { Option } = Select;
+  const descriptionIfa = (
+    <div>
+      <p>IFA adds Togawa, Albion, Innovative,</p>
+      <p>and Militant to the base selection</p>
+    </div>
+  );
+  const descriptionBase = (
+    <div>
+      <p>Base game includes the five default</p>
+      <p>faction and mat combinations</p>
+    </div>
+  );
+  const descriptionHi = (
+    <div>
+      <p>Hi-tier removes Albion, Togawa,</p>
+      <p>Agricultural, and Engineering</p>
+    </div>
+  );
+  const descriptionLo = (
+    <div>
+      <p>Lo-tier removes Rusviet, Crimea,</p>
+      <p>Innovative, and Militant</p>
+    </div>
+  );
+
```

Comment:
Personally, I think it would be best if all of this info was shown in the beginning by default, so a first-time user knows what's up, instead of having to switch through each of the options that they probably don't understand to then figure out what they mean.